src/tribol/utils/TestUtils.cpp

```diff
@@ -94,28 +94,32 @@ TestMesh::~TestMesh()
 }
 
 //------------------------------------------------------------------------------
-void TestMesh::clear()
+void TestMesh::clear( bool keepCoords )
 {
-   if (this->mfem_mesh != nullptr)
```

Comment:
I'm thinking we should keep `this->` to be consistent with the way we address member variables in this method.  More generally, it might be good to have a project-wide convention for naming and addressing member variables, parameters, etc.

Reply:
Yes. I think that could be a straightforward refactor. The convention I'm used to is that member variables are m_<name>, and when you see that you don't need this->. I've probably used this-> too much, but for this class there is no 'm_' so you can't easily tell. Let's refactor that in another cleanup PR.